iOS Client/Store/XMLParsingDelegate.swift

```diff
@@ -19,6 +19,14 @@ class PodcastXMLParsingDelegate: NSObject, XMLParserDelegate {
         self.foundCharacters += string
     }
     
+    func parser(_ parser: XMLParser, didStartElement elementName: String, namespaceURI: String?, qualifiedName qName: String?, attributes attributeDict: [String : String]) {
+        if elementName == "enclosure" {
+            if let url = attributeDict["url"] {
```

Comment:
you can one line this with `if elementName == "enclosure", let url = attributeDict["url"] { . . . }` I think . . . Certainly can do it if you're willing to put the let clause first

Reply:
Question: What's the point of this? Wouldn't this just get overwritten below by the `guid` case of `didEndElement`?

Reply:
Yeah so this is a little artifact that the rss feeds are not structured the same way. 2 of the feeds have the url as fields on the xml element, and one has its own element called `guid` with the url. So the feeds that have `enclosure` don't have `guid` so they don't get overwritten. I think we might need to do some reexamination of this logic but I think it's good to get in for now

Reply:
ah okay, seems reasonable. And I see you only have access to this `attributeDict` on the `didStartElement` delegate method.